components/remote_ros_connectivity.mdx

```diff
@@ -0,0 +1,52 @@
+To use ROS desktop tools, you will need the Remote PC to be able to connect to your robot's ROS master. This will allow
+you to run ROS commands like `rostopic list`, `rostopic echo`, `rosnode list`, and others, from the Remote PC and the
+output will reflect the activity on your robot's ROS master, rather than on the Remote PC. This can be a tricky
+process, but we have tried to make it as simple as possible.
+
+In order for the ROS tools on the Remote PC to talk to your robot, they need to know two things:
+
+- How to find the ROS master, which is set in the `ROS_MASTER_URI` environment variable, and
+- How processes on the ROS master can find the Remote PC, which is the `ROS_IP` environment variable.
+
+The suggested pattern is to create a file in your home directory called `remote-robot.sh` with the following contents:
+
+```bash
+export ROS_MASTER_URI=http://cpr-robot-0001:11311 # Your robot's hostname
+export ROS_IP=10.25.0.102 # Your Remote PC's wireless IP address
+```
+
+If your network does not already resolve your robot's hostname to its wireless IP address, you may need to add a
+corresponding line to the Remote PC's `/etc/hosts` file:
+
+```bash
+10.25.0.101 cpr-robot-0001
+```
+
+NOTE: You can verify the hostname and IP address of your robot using the following commands during an SSH session
```

Comment:
Consider using an Admonition.
You can add Admonitions inside an expand section.
(Refer to the Husky Mainenance page as an example)

Reply:
Done